### PR TITLE
frontend of homepage, register and log in

### PR DESCRIPTION
**Pull request by NaomiW7**

### CODE REVIEW

File: src/pages/Profile/editProfiel.js
Comment (yunchipang) [inline src/pages/Profile/editProfiel.js:1]: file name has typo.` editProfiel.js` -> `editProfile.js`

**Review by yunchipang — CHANGES_REQUESTED** [review]
remove all git tracked`.idea/` files by adding `.idea/` to `.gitignore`

File: src/App.js [hunk 1]
```diff
@@ -38,6 +40,8 @@ function App() {
         <Route path="/seller-portal" element={<SellerPortal />} />
         <Route path="/buyer-portal" element={<BuyerPortal />} />
         <Route path="/properties" element={<PropertyList />} />
+        <Route path="/homepage" element={<HomePage />}/>
```
Comment (yunchipang) [inline src/App.js]: could you use `/` instead of `/homepage`? it makes more sense this way.
otherwise the root of our website it empty.

**Review by yunchipang — CHANGES_REQUESTED** [review]

**Comment by NaomiW7** [comment]
> remove all git tracked`.idea/` files by adding `.idea/` to `.gitignore`

Done

File: src/App.js [hunk 2]
```diff
@@ -1,11 +1,15 @@
 import React, { useEffect, useState } from "react";
-import { BrowserRouter as Router, Route, Routes } from "react-router-dom";
+import { BrowserRouter as Router, Route, Routes} from "react-router-dom";
 import "./App.css";
 import SellerPortal from "./pages/SellerPortal";
 import BuyerPortal from "./pages/BuyerPortal";
 import PropertyList from "./pages/PropertyList";
+import HomePage from "./pages/HomePage";
+import SignupLogin from "./pages/SignupLogin";
+import Profile from "./pages/Profile";
```
Comment (yunchipang) [inline src/App.js:9]: remove this line

File: src/App.js [hunk 3]
```diff
@@ -38,6 +42,10 @@ function App() {
         <Route path="/seller-portal" element={<SellerPortal />} />
         <Route path="/buyer-portal" element={<BuyerPortal />} />
         <Route path="/properties" element={<PropertyList />} />
+        <Route path="/" element={<HomePage />}/>
+        <Route path="/signuplogin" element={<SignupLogin />}/>
+        <Route path="/logout" element={<Logout />}/>
+        <Route path="/profile" element={<Profile />}/>
```
Comment (yunchipang) [inline src/App.js]: remove this line if you're giving up on all Profile related features in this PR.

**Review by yunchipang — CHANGES_REQUESTED** [review]
now throws error because profile is hidden but is still referred in `App.js`

**Review by yunchipang — APPROVED** [review]

**Pull request merged**